### ISSUE TITLE
Issue with Uploading images/obj.

### ISSUE THREAD

**Issue by RaliiTheAvali** [Neos-Metaverse/NeosPublic#3114]
### Describe the bug?

When others try to upload an image into a world ( whether freshly created, or an established world) it appears that eventually after either a given time frame or upload count, that images or objects just simply get stuck and refuse to load for anyone except the person who uploaded the said item into the world.

Attempts to pinpoint the cause of the issue have been inconsistent as some times we can get up to about 10 uploads before others report that images do not load and get stuck in the debug menu and sometimes it can happen after just the 1st upload.

Upload speed ( rate how how many images or objects are being loaded at once) does not seem to affect frequency or speed of when the issue props up. as we have tried doing them one at a time wait till every one in the instance sees its loaded them on to the next.

A serious side effect is that when the issue happens it can affect de-sync between players, the ability to save the world, the ability to save an object, and in rare situations a full Sync error.

Only way to solve the issue is to launch a new world till the issue props up again ( sometimes only taking minuets), or restarting neos.

### Relevant issues

NA

### To Reproduce

1. Get 1-3 ppl into an instance ( preferably with variable net speeds and one who has maybe less than 5MBPS Upload) ( unsure if net speed affects this)
2. Have a user upload images or objects
3. Eventually an item should get stuck and not load at all. ( De-bug will show an item in waiting but with -1 or 0Bits

### Expected behavior

(still somewhat confused on what this means, does this mean " what do you expect to happen if the bug is resolved?" or is this a " what you expect to happen if the Recreation above is followed?")

### Log Files

Unable to find logs/No log was created as of recent to show the Sync Error i had yesterday.

### Screenshots

![20210929221945_1](https://user-images.githubusercontent.com/75050016/135539627-eccff2aa-fb51-4865-8a9c-c617b98fc236.jpg)
![20210914221543_1](https://user-images.githubusercontent.com/75050016/135539653-c423262a-15fd-4e88-9c79-f64212bd9e06.jpg)

### How often does it happen?

Always

### Does the bug persist after restarting Neos?

Yes

### Neos Version Number

2021.9.29.842

### What Platforms does this occur on?

Windows

### Did this work before?

Yes

### If it worked before, on which build?

Unknown

### Additional context

Unknown on which build due to a hiatus from Neos for at least 1 month. 

### Reporters

Ralii#2181

**Comment by RaliiTheAvali** [comment]
Would like to point out, the one item that said 3.17 MB at 24% got stuck and did not go any further. ( related to Screen shot above)

**Comment by ProbablePrime** [comment]
Could this be related to #2062

**Comment by RaliiTheAvali** [comment]
> 
> 
> Could this be related to #2062

Yes ( wasn't aware there was one, plus Apr. is pretty far back) but i feel this issue has only gotten worse since then as i can't seem to go past 15-30 min in a single world before that happens. and i could only have been online fore that 15-30 min span.

**Comment by Frooxius** [comment]
Yeah this seems like same issue as #2062 

Do you have a log file from such case? Ideally from both ends (receiving user and sending)? If this can be replicated in clean way and logs gathered that can help. Without those it's going to be difficult to address this.
